Makefile

```diff
@@ -17,6 +17,11 @@ export GO111MODULE = on
 # process build tags
 
 build_tags = netgo
+
+ifeq ($(STABLE),true)
```

Comment:
I rarely see the `STABLE` flag out there, is it something known?

An alternative would be just to have 2 scripts:
- `make build` -> stable
- `make build-experimental` -> experimental

Don't want to block the PR on this, just a suggestion.

Reply:
Not really, I didn't see anyone using STABLE flag either.  My intention was not to introduce any extra commands to handle this. That's the reason to use an ENV setting. But later looks self-explanatory.
Also might need to have 2 install commands too if we follow separate commands to generate stable and experimental builds

Reply:
I'm fine with env variable, but my preference would be for having the name of the flag `EXPERIMENTAL`, `ENABLE_EXPERIMENTAL_FEATURES`, or `EXPERIMENTAL_BUILD`, etc. (with defualt to false), rather than `STABLE` with default to true. I like this better because "stable" usually implies things like API guarantees, and until we reach regen ledger v1.0, or v1 proto definitions, we don't really have such guarantees around stability of our API yet. "Experimental" speaks more about the featureset, which can include which modules are part of a build.

Not to bike shed, but thats my 2c.

Reply:
Agree about experimental vs stable. Otherwise looks good!

Reply:
Agree, will change it to `EXPERIMENTAL`